tools/ports/sdl2_image.py

```diff
@@ -52,6 +54,11 @@ def create(final):
     if 'jpg' in settings.SDL2_IMAGE_FORMATS:
       defs += ['-sUSE_LIBJPEG']
 
+    if settings.USE_PTHREADS:
+      defs += ['-sUSE_PTHREADS']
+    if settings.WASM_WORKERS:
+      defs += ['-sWASM_WORKERS']
```

Comment:
![image](https://user-images.githubusercontent.com/1687930/212498490-1b86c3a9-8a39-4b3c-a1b5-6e3e62a7c778.png)


I think this will only work if I also changed the sdl2.py get_lib_name .... 🤔
from 
```
def get_lib_name(settings):
  return 'libSDL2' + ('-mt' if settings.USE_PTHREADS else '') + '.a'
```
to 
```
def get_lib_name(settings):
  return 'libSDL2' + ('-mt' if settings.SHARED_MEMORY else '') + '.a'
```